lib/pharos/phases/validate_host.rb

```diff
@@ -50,10 +52,18 @@ def validate_unique_hostnames
         raise Pharos::InvalidHostError, "Duplicate hostname #{@host.hostname} for hosts #{duplicates.map(&:address).join(',')}"
       end
 
+      def check_installed
+        %w(ip grep curl uname hostname head cut sed).each do |cmd|
+          logger.debug { "Checking if command '#{cmd}' exists" }
+          raise Pharos::InvalidHostError, "Host does not seem to have the required binary '#{cmd}' installed." unless transport.exec?("command -v \"#{cmd}\" &> /dev/null")
```

Comment:
I think we should just install those packages. Configurer could have `#install_prereqs` or similar method that we could call before we start to validating more deeply.

Reply:
Installing packages during validation phase sounds a bit wrong.

Reply:
@jnummelin WDYT

Reply:
Perhaps with something like `--install-dependencies`?

Reply:
Yeah, we should just install all the needed "tools". But maybe keep these tools installations bit separate from others as these are more for validation purposes and as @kke said, not optimal to install stuff on validation phase.

Reply:
Also many of these commands are used already before this validation happens.